features/base/conference/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { CONNECTION_DISCONNECTED } from '../connection';
+import { ReducerRegistry } from '../redux';
+
+import {
+    CONFERENCE_JOINED,
+    CONFERENCE_LEFT
+} from './actionTypes';
+
+/**
+ * Listen for actions that contain the conference object, so that
+ * it can be stored for use by other action creators.
+ */
+ReducerRegistry.register('features/base/conference', (state = null, action) => {
+    switch (action.type) {
+    case CONFERENCE_JOINED:
+        return action.conference;
+
+    case CONNECTION_DISCONNECTED:
+    case CONFERENCE_LEFT:
```

Comment:
I don't like it that both CONNECTION_DISCONNECTED and CONFERENCE_LEFT clear the features/base/conference state. To me it means that we haven't defined a sequence of conference lifecycle events. I believe it's very important that we define the sequence of conference lifecycle events first and then write source code.

For example. one could choose to define that CONNECTION_DISCONNECTED will cause CONFERENCE_LEFT if the local participant is still joined in the conference. The reasoning for such a choice could be that the conference cannot be considered joined if the connection is disconnected.